### PR TITLE
Change history logic

### PR DESCRIPTION
**Pull request by raressiteavu**

### CODE REVIEW

**Review by mereualexandra — APPROVED** [review]
I would add another test just to be on the safe side: 
create report + update comment + update report + update comment again => 4 history events
Overall, great work! 😎

**Comment by sorinstelian** [comment]
> I would add another test just to be on the safe side: create report + update comment + update report + update comment again => 4 history events Overall, great work! 😎

@mereualexandra I think we also need to add (from backend) a history entry when commenting. But not in this PR.

**Review by mereualexandra — APPROVED** [review]
Great work 😎

File: core/api/views/cp_records.py [hunk 1]
```diff
@@ -290,6 +291,20 @@ def _get_new_cp_records(self, cp_report):
             "remarks": cp_report.comment,
         }
 
+        history = []
```
Comment (sorinstelian) [inline core/api/views/cp_records.py:294]: This looks good, but I would move it to a method (or two) just like all other sections are implemented, to keep this method nice and slim.

**Pull request merged**